### ISSUE TITLE
multi option to disable stategies

### ISSUE THREAD

**Issue by thistleknot** [enzoampil/fastquant#349]
I like the OR structure

However.  I'd also like to see if some strategies are excluded altogether.

If there was a strat_opts where the values were blank that would disable the strategy that would be nice.

ex.

bbands has ,20 meaning that at least on one of the grid searchs.  the bbands strategy is completely disabled while the rest are enabled (in this example)

Something like that.

strats_opt = { 
    "smac": {"fast_period": [35, 50], "slow_period": [100, 200]}, 
    "rsi": {"rsi_lower": [15, 30], "rsi_upper": [70, 85]},
    "macd": {"fast_period": [12], "slow_period": [26], "signal_period": [9], "sma_period": [30], "dir_period": [10]},
    "bbands": {"period": [,20], "devfactor": [,2.0]},
    
    #"ema": {"fast_period": [9,10,12], "slow_period": [30, 40, 50]},     
    "custom": {"upper_limit": [1.5], "lower_limit":[-1.5]}
}

**Comment by enzoampil** [comment]
Good Idea, we can add this to the feature backlog - thanks @thistleknot !
